src/components/Card/CardReadMore.js

```diff
@@ -1,28 +1,46 @@
 import React from 'react'
 import PropTypes from 'prop-types'
 import classNames from 'classnames'
+import { Route, Link, BrowserRouter as Router } from 'react-router-dom'
 
 import Icon from '../Icon/Icon'
 
+const tagPropType = PropTypes.oneOfType([
+  PropTypes.func,
+  PropTypes.string,
+  PropTypes.shape({ $$typeof: PropTypes.symbol, render: PropTypes.func }),
+  PropTypes.arrayOf(PropTypes.oneOfType([
+    PropTypes.func,
+    PropTypes.string,
+    PropTypes.shape({ $$typeof: PropTypes.symbol, render: PropTypes.func }),
+  ]))
+]);
+
 const propTypes = {
   className: PropTypes.string,
   iconName: PropTypes.string,
   href: PropTypes.string,
-  text: PropTypes.string
+  text: PropTypes.string,
+  tag: tagPropType,
+  to: PropTypes.string
 }
 
 const defaultTypes = {
-  iconName: 'it-arrow-right'
+  iconName: 'it-arrow-right',
+  tag: 'a'
 }
```

Comment:
create `defaultProps` here. Also, `iconName` shouldn't be handled here, its already in props